_episodes/01-connecting.md

```diff
@@ -99,7 +100,7 @@ mentioned above before moving on to navigating the remote system.
 ## Logging onto the system
 
 With all of this in mind, let's connect to a remote HPC system. In this workshop, we will connect to
-{{ site.workshop_host }} - an HPC system located at the {{ site.workshop_host_location}}. Although it's unlikely
+{{ site.workshop_host }} --- an HPC system located at the {{ site.workshop_host_location}}. Although it's unlikely
```

Comment:
Since we're editing this line, please substitute 
`{{ site.workshop_host_location }}` for `{{ site.workshop_host_location}}` (liquid variables should be padded with spaces). Thanks!